### PR TITLE
KTIJ-23097 "Move Statement down" breaks out of scope when skipping over comment line

### PR DESCRIPTION
**Pull request by t-kameyama**
https://youtrack.jetbrains.com/issue/KTIJ-23097

### CODE REVIEW

File: plugins/kotlin/idea/src/org/jetbrains/kotlin/idea/codeInsight/upDownMover/KotlinExpressionMover.kt [hunk 1]
```diff
@@ -503,7 +506,7 @@ class KotlinExpressionMover : AbstractKotlinUpDownMover() {
 
             val movableElement = element.getParentOfTypesAndPredicate(
                 strict = false,
-                parentClasses = *MOVABLE_ELEMENT_CLASSES,
```
Comment (t-kameyama) [inline plugins/kotlin/idea/src/org/jetbrains/kotlin/idea/codeInsight/upDownMover/KotlinExpressionMover.kt:506]: Redundant spread operator

File: plugins/kotlin/idea/src/org/jetbrains/kotlin/idea/codeInsight/upDownMover/KotlinExpressionMover.kt [hunk 2]
```diff
@@ -503,7 +506,7 @@ class KotlinExpressionMover : AbstractKotlinUpDownMover() {
...
+                parentClasses = MOVABLE_ELEMENT_CLASSES,
```
Comment (t-kameyama) [inline plugins/kotlin/idea/src/org/jetbrains/kotlin/idea/codeInsight/upDownMover/KotlinExpressionMover.kt:509]: Redundant spread operator

**Comment by abelkov** [comment]
Can you please support a related case. On "move up", when there are multiple comment lines above, Java skips over exactly one line. But Kotlin skips all comments. I think Java behavior is better and more consistent with "move down".

```
public class JJJ {
    public static void main(String[] args) {
        // Comment
        // Comment
        String s = "";
        // Comment
    }
}
```

**Comment by vanniktech** [comment]
@abelkov 100% agreed. I also encountered this yesterday but I haven't had the time to comment this. (I'm the one who created the Youtrack issue)

**Comment by t-kameyama** [comment]
> Can you please support a related case. On "move up", when there are multiple comment lines above, Java skips over exactly one line. But Kotlin skips all comments. I think Java behavior is better and more consistent with "move down".

[674aa47](https://github.com/JetBrains/intellij-community/pull/2185/commits/674aa472a272a77a379840b0fe6ef080a242f657)

**Review by abelkov — APPROVED** [review]

**Pull request closed without merging**